cmd/yurt-manager/app/options/hubleaderconfigcontroller.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2025 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the License);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an AS IS BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/openyurtio/openyurt/pkg/yurtmanager/controller/hubleaderconfig/config"
+)
+
+type HubLeaderConfigControllerOptions struct {
+	*config.HubLeaderConfigControllerConfiguration
+}
+
+func NewHubLeaderConfigControllerOptions(hubleaderNamespace string) *HubLeaderConfigControllerOptions {
+	return &HubLeaderConfigControllerOptions{
+		&config.HubLeaderConfigControllerConfiguration{
+			ConcurrentHubLeaderConfigWorkers: 3,
+			HubLeaderNamespace:               hubleaderNamespace,
```

Comment:
we can use `config.GenericConfiguration.WorkingNamespace` directly instead of defining a new one.